include/freerdp/server/rdpgfx.h

```diff
@@ -21,6 +21,7 @@
 #define FREERDP_CHANNEL_RDPGFX_SERVER_RDPGFX_H
 
 #include <freerdp/channels/rdpgfx.h>
+#include <freerdp/freerdp.h>
```

Comment:
Need to ensure this is legit

Reply:
it wasn't there, and doesn't make sense

Reply:
I'm not sure it compiles without it tho, can you please test that?